conjur-oss/values.yaml

```diff
@@ -34,6 +34,7 @@ database:
   # 64-character alphanumeric password will be auto-generated.
   # Note that this value will be ignored for Helm upgrade operations.
   password: ""
+  remotePassword: ""
```

Comment:
Just on the base of reading comment, If remotePassword will be filled, is password also be with random alphanumeric password?

Reply:
remotePassword is going to be filled only for cases that postgres is external - and then password field value is irrelevant

Reply:
Yes I understand that remotePassord will be filled only if set by looking at:
`{{ if .Values.database.remotePassword }}`
What I claim is, leaving password empty will lead to creation of auto-generate password, leading to bad UX ("two passwords fields, which one was used? I only wanted to remotePassword") and to a minor security issue of no manual cleaning / extra password in space - see "When is Manual Deletion Required?" column in readme.
+
https://github.com/cyberark/conjur-oss-helm-chart/blob/59bac7069f9b9cc06991eeca9d0f309321f9a95a/conjur-oss/templates/secrets.yaml#L89
with wrong database url

Reply:
There should be a blank line between the `password` and `remotePassword` entry to make it clear that the "alphanumeric password will be auto-generated" comment doesn't apply to `remotePassword'.

Also, `remotePassword` should have a comment explaining how it is used.

Reply:
`database.remotePassword` will need a validity check added to `conjur-oss/values.schema.json`, similar to what's there for `database.password`.